ui/components/component-library/button-link/button-link.js

```diff
@@ -36,6 +35,7 @@ export const ButtonLink = ({
       iconLoadingProps={{
         size: size === BUTTON_LINK_SIZES.INHERIT ? SIZES.INHERIT : SIZES.MD,
       }}
+      {...props}
     />
```

Comment:
Small update to ButtonLink to allow for text customization

Fixes: https://github.com/MetaMask/metamask-extension/issues/17477